src/client/app/needs/config.js

```diff
@@ -79,6 +79,20 @@
                         content: '<i class="fa fa-lock"></i> Admin'
                     }*/
                 }
+            },
+            {
+                state: 'searchcompletedneeds',
```

Comment:
your state should be refactored to have an abstract one, an index one and yours.
Contact Mykola to find out the details.